src/SELF_DGModel3D_t.f90

```diff
@@ -458,15 +455,15 @@ subroutine Read_DGModel3D_t(this,fileName)
     integer(HID_T) :: solOffset(1:5)
     integer :: firstElem
 
-    if(this%decomp%mpiEnabled) then
+    if(this%mesh%decomp%mpiEnabled) then
       call Open_HDF5(fileName,H5F_ACC_RDWR_F,fileId, &
-                     this%decomp%mpiComm)
+                     this%mesh%decomp%mpiComm)
```

Comment:
Same as with dgmodel2d; we need single and multi-domain tests that load pickup files so that we can cover serial and parallel reads.

Reply:
Resolved with [4f7a7a0](https://github.com/FluidNumerics/SELF/pull/53/commits/4f7a7a0e157b85dc677092583d328816442ade81)